.config/smoldot.dic

```diff
@@ -0,0 +1,144 @@
+143
+ABI
+API
+APIs
+BIP39
+BLAKE2
+Base58
+CLI
+Deserialize/S
+DoS
+ECDH
+Ed25519
+Endian
+Enqueue/S
+FFI
+FNV
+GrandPa
+HMAC
+HashDoS
+IP
+JIT
+JITs
+JSON
+Jaeger
+Kademlia
+Kusama
+LEB128
+LLVM
+LRU
+MERCHANTABILITY
+Merkle
+NPM
+Namespace
+POV
+Polkadot
+PolkadotJS
+Protobuf
+QUIC
+RPC
+SHA-256
+SQLite
+SS58
+Sr25519
+TCP
+TODO
+Trie
+UDP
+UI/S
+UTC
+Unencrypt/D
+VM
+VM's
+VRF
+Wasi
+Wasm
+WebAssembly
+WebSocket/M
+Westend
+Yamux
+aborter
+allocator
+amongst
```

Comment:
Isn't this a common English word? It's weird

Reply:
missing here https://github.com/drahnr/cargo-spellcheck/blob/master/hunspell-data/en_US.dic